mllib/src/main/scala/org/apache/spark/ml/regression/GeneralizedLinearRegression.scala

```diff
@@ -592,6 +629,59 @@ object GeneralizedLinearRegression extends DefaultParamsReadable[GeneralizedLine
   }
 
   /**
+    * Tweedie exponential family distribution.
+    * The default link for the Tweedie family is the log link.
+    */
+  private[regression] object Tweedie extends Family("tweedie") {
+
+    val defaultLink: Link = Log
+
+    var variancePower: Double = 1.5
```

Comment:
Would you please suggest a better way to set the variancePower? I want to be consistent with the existing code to have the `Family` objects, but I need to also pass on the input `variancePower` to the `Tweedie` object which is used to compute the variance function. Any suggestion will be highly appreciated. @srowen @yanboliang